### PR TITLE
fix entitytype and add beta

### PR DESCRIPTION
**Pull request by knightsu**

### CODE REVIEW

File: concepts/search-concept-messages.md [hunk 1]
```diff
@@ -148,4 +148,4 @@ Authorization: Bearer AAD_TOKEN
 
 Find out more about:
 
-- The Microsoft Search [Query API](/graph/api/search-query?view=graph-rest-beta)
+- The Microsoft Search [Query API](/graph/api/search-query?view=graph-rest-betasear)
```
Comment (nmoreau) [inline concepts/search-concept-messages.md]: Is this correct ? Betasear?

File: api-reference/beta/resources/searchrequest.md [hunk 2]
```diff
@@ -40,7 +42,7 @@ The following is a JSON representation of the resource.
 {
   "stored_fields": ["String"],
   "contentSources": ["String"],
-  "entityTypes": ["String"],
+  "entityTypes": "String",
```
Comment (nmoreau) [inline api-reference/beta/resources/searchrequest.md]: This should be a collection of entity Types

File: api-reference/beta/resources/searchrequest.md [hunk 3]
```diff
@@ -18,7 +20,7 @@ The search request to be sent to the query endpoint. It contains the type of ent
 |stored_fields|String collection |Contains the fields to be returned for earch _so urces object. Note this is only applicable when entityType=`microsoft.graph.externalItem` is specified in the response.|
 |contentSources|String collection|Contains the connection to be targeted. <br>Respect the following format : `/external/connections/connectionid` where `connectionid` is the ConnectionId been defined in the Connectors Administration <br> Note contentSource is only applicable when entityType=`microsoft.graph.externalItem`. |
 |enableTopResults|Boolean|This triggers relevance sorting for messages <br> This is only applicable for entityType=`microsoft.graph.message`.|
-|entityTypes|String collection| Possible values are: `microsoft.graph.event`, `microsoft.graph.message`, `microsoft.graph.driveItem`, `microsoft.graph.externalFile`, `microsoft.graph.externalItem`.|
+|entityTypes|entityTypes| Possible values are: `microsoft.graph.event`, `microsoft.graph.message`, `microsoft.graph.driveItem`, `microsoft.graph.externalFile`, `microsoft.graph.externalItem`.|
```
Comment (nmoreau) [inline api-reference/beta/resources/searchrequest.md]: Did you follow the instructions from the doc guidance on enums ? Or do you plan to make it in another round ?

File: api-reference/beta/resources/searchrequest.md (hunk 2 above)
Comment (knightsu) [reply api-reference/beta/resources/searchrequest.md]: based on the doc https://msgo.azurewebsites.net/add/document/guidelines/enumerations.html
it should be string pls confirm with angela if i'm wrong

File: api-reference/beta/resources/searchrequest.md (hunk 3 above)
Comment (knightsu) [reply api-reference/beta/resources/searchrequest.md]: i follow the doc guidance

File: api-reference/beta/resources/searchrequest.md (hunk 2 above)
Comment (nmoreau) [reply api-reference/beta/resources/searchrequest.md]: Can you please check with her.

**Pull request merged**